invenio_records_lom/ext.py

```diff
@@ -40,6 +40,16 @@ def init_config(self, app):
         Override configuration variables with the values in this package.
         """
         for k in dir(config):
+            if k == "LOM_PERSISTENT_IDENTIFIER_PROVIDERS":
```

Comment:
why is it necessary to update those two directly?

Reply:
Other modules (e.g. `invenio_moodle`) might want to add to these settings.
However, the order in which invenio-modules are loaded isn't guaranteed to stay the same.
Hence the "more obvious" solutions don't work:
- `config["KEY"] = attr` would overwrite other modules, if another module is executed first
- `config.setdefault("KEY", attr)` would fail to update if another module populates `config["KEY"]` first